tool/plugin/lib/plugin.dart

```diff
@@ -241,7 +253,7 @@ Future<bool> performReleaseChecks(ProductCommand cmd) async {
           log('the .travis.yml file needs updating: plugin generate');
         }
       } else {
-        log('the current git branch must be named "release_${cmd.releaseMajor}"');
+        log('the current git branch must be named "$expectedName"');
```

Comment:
Do we need to fail the build here?

Reply:
It does fail. The function returns false, which causes failure.